scripts/extensions/pool_update.precheck

```diff
@@ -125,22 +124,24 @@ def execute_precheck(session, control_package, yum_conf_file, update_precheck_fi
                         prerequisite_updates += prerequisite + ' '
                     raise PrerequisiteMissing(prerequisite_updates.rstrip())
                 else:
-                    raise PrecheckFailure(precheckfailure_msg)
+                    raise PrecheckFailure(errmsg)
             else:
-                raise PrecheckFailure(precheckfailure_msg)
+                raise PrecheckFailure(errmsg)
         else:
-            raise PrecheckFailure()
+            raise PrecheckFailure(output)
     else:
```

Comment:
I think this `else:` is redundant: all code paths of the `if p.returncode != 0:` result in an exception being thrown.

Can you remove it and pull the indentation of the rest of the function in one level.